### PR TITLE
Created Minneapolis website

### PR DESCRIPTION
**Pull request by AverseABFun**

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/minneapolis**](https://counterspell-1f58ju0ab.hackclub.dev/minneapolis) |
|:-:|
|<a href="https://counterspell-1f58ju0ab.hackclub.dev/minneapolis">Screenshot of /minneapolis</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/oCMv57ce3rAhAA2Pi7NwfsZ_gznP7g8gd-Q1JZFYCRXOaHssj7BNzmDJTvRVJqJHwWBP3QBezaX9cRhOdgmHyg.png">(view full size)</a> |







Commit 3a4c3d6da69a14bd9acf98a58258640a35179822 (https://counterspell-1f58ju0ab.hackclub.dev).

**Review by samdev-7 — APPROVED** [review]

File: components/Minneapolis/Subscribe.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    fetch(form.action, {
+      method: "POST",
+      body: new URLSearchParams(new FormData(form)),
+    })
+      .then((response) => {
+        if (response.ok) {
+          form.reset();
+          setSubmitBtn("Success, redirecting");
+          setTimeout(()=>{
+            document.location.href = "https://wa.tcmaker.org/event-5908977";
```
Comment (samdev-7) [inline components/Minneapolis/Subscribe.js:46]: I'd recommend against a redirect, as it's confusing to the user why the site changed. A button may be a better option.

Let me know if you want to change this or keep it as is! @AverseABFun

**Comment by AverseABFun** [comment]
Hm, the thing is that it needs to be on both our lists and I put text
informing them.

**Comment by samdev-7** [comment]
Cool, thanks! I'll merge this for now, open another PR if changes need to be made.

**Pull request merged**